TestCases/hybrid_regression.py

```diff
@@ -253,11 +253,20 @@ def main():
     axi_rans_air_nozzle           = TestCase('axi_rans_air_nozzle')
     axi_rans_air_nozzle.cfg_dir   = "axisymmetric_rans/air_nozzle"
     axi_rans_air_nozzle.cfg_file  = "air_nozzle.cfg"
-    axi_rans_air_nozzle.test_iter = 10
-    axi_rans_air_nozzle.test_vals         = [-12.093553, -6.630383, -8.798737, -2.399130, -1938.200000]
+    axi_rans_air_nozzle.test_iter = 50
+    axi_rans_air_nozzle.test_vals         = [-3.896930, 1.677006, 0.241104, 4.719832, -303.650000]
     axi_rans_air_nozzle.test_vals_aarch64 = [-12.093539, -6.630357, -8.798732, -2.399130, -1938.200000]
     test_list.append(axi_rans_air_nozzle)
 
+    # Axisymmetric air nozzle (transonic)
+    axi_rans_air_nozzle_restart           = TestCase('axi_rans_air_nozzle_restart')
+    axi_rans_air_nozzle_restart.cfg_dir   = "axisymmetric_rans/air_nozzle"
+    axi_rans_air_nozzle_restart.cfg_file  = "air_nozzle_restart.cfg"
+    axi_rans_air_nozzle_restart.test_iter = 10
+    axi_rans_air_nozzle_restart.test_vals         = [-4.333049, 1.244961, -0.175781, 4.228753, -1515.600000]
+    axi_rans_air_nozzle_restart.test_vals_aarch64 = [-12.093539, -6.630357, -8.798732, -2.399130, -1938.200000]
+    test_list.append(axi_rans_air_nozzle_restart)
```

Comment:
We are not going to update restart cases like this

Reply:
Clear. For the restart, I suggest we always converge the cases to the convergence criterion that is in the config file, so it is consistent and easy to update the restart.

Reply:
Makes sense.
For the record I think your idea for the restart is good for testing _the restart_ but we would need to run N + M in the first run, restart at N and measure at M (with reference values obtained automatically from N+M).
It's just that this doesn't cover the correctness of the solution, unless the first run is all the way to convergence, which we should do for small-enough cases.